projects/packages/videopress/src/client/admin/components/checkbox/stories/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from 'react';
+import Checkbox from '..';
+import type { ComponentStory, ComponentMeta } from '@storybook/react';
+
+export default {
+	title: 'Packages/VideoPress/Checkbox',
+	component: Checkbox,
+	parameters: {
+		layout: 'centered',
+	},
+} as ComponentMeta< typeof Checkbox >;
+
+const Template: ComponentStory< typeof Checkbox > = args => {
+	const [ checked, setChecked ] = useState( false );
+	const onChange = current => setChecked( current );
+	return <Checkbox { ...args } checked={ checked } onChange={ onChange } />;
```

Comment:
You can use `updateArgs` to sync with the storybook `args` if you decide to add `checked` to the controls: https://github.com/Automattic/jetpack/blob/trunk/projects/packages/videopress/src/client/admin/components/pagination/stories/index.tsx#L20

Reply:
Thanks! Did it.